openedx/core/djangoapps/credit/api.py

```diff
@@ -111,6 +111,37 @@ def get_credit_requirements(course_key, namespace=None):
     ]
 
 
+def get_credit_requirement(course_key, namespace, name):
+    """Returns the requirement of a given course, namespace and name.
+
+    Args:
+        course_key(CourseKey): The identifier for course
+        namespace(str): Namespace of requirement
+        name: Name of the requirement
+
+    Returns:
+        'CreditRequirement' object
+
+    """
+    return CreditRequirement.get_course_requirement(course_key, namespace, name)
+
+
+def set_credit_requirement_status(user_name, requirement, status="satisfied", reason=None):
+    """Update Credit Requirement Status for given username and requirement
+        if exists else add new.
+
+    Args:
+        user_name(str): Username of the user
+        requirement(CreditRequirement): 'CreditRequirement' object
+        status(str): Status of the requirement
+        reason(dict): Reason of the status
+
+    """
+    CreditRequirementStatus.add_or_update_requirement_status(
```

Comment:
The underlying class method returns `requirement_status`. I'd at least return that.